azureadps-2.0/AzureAD/AzureActiveDirectory.md

```diff
@@ -10,6 +10,11 @@ ms.custom: iamfeature=PowerShell
 ---
 # Azure Active Directory PowerShell for Graph module
 ## Description
+
+>[!IMPORTANT]
+> Azure AD PowerShell is planned for deprecation. See the [deprecation update](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/azure-ad-change-management-simplified/ba-p/2967456) for more details on the deprecation plans. We recommend that you start trying [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview). Microsoft Graph PowerShell is the replacement for Azure AD PowerShell and is recommended for interacting with Azure AD. See the [migration FAQ](../../docs-conceptual/azureadps-2.0/migration-faq.yml) for answers to frequent migration queries.
```

Comment:
```suggestion
> Azure AD PowerShell is planned for deprecation. See the [deprecation update](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/azure-ad-change-management-simplified/ba-p/2967456) for more details on the deprecation plans. We recommend that you start trying [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview). Microsoft Graph PowerShell is the replacement for Azure AD PowerShell and is recommended for interacting with Azure AD. See the [migration FAQ](../../docs-conceptual/azureadps-2.0/migration-faq.yml) for answers to frequent migration queries.
```
```suggestion
> Azure AD PowerShell is planned for deprecation. See the [deprecation update](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/azure-ad-change-management-simplified/ba-p/2967456) for more details on the deprecation plans. We recommend that you start trying [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview). Microsoft Graph PowerShell is the replacement for Azure AD PowerShell and is recommended for interacting with Azure AD. See the [migration FAQ](../../docs-conceptual/azureadps-2.0/migration-faq.yml) for answers to frequent migration queries.
```

@mswaweru - Can you please reword the last 3 sentences. There's a lot of passive voice and would be great to see a better description of what the value prop and benefits are for customers using Azure AD PS now and why they need to use MS Graph PS.